<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QWidget" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="connectServer">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendServer">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBrowser">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>清除聊天记录</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>240</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>0</y>
     <width>325</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入要连接的ip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入要连接的端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progress">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>查看处理进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
